src/services/EvalFactor.jl

```diff
@@ -182,13 +182,13 @@ function computeAcrossHypothesis!(ccwl::Union{<:CommonConvWrapper{F},
       for i in 1:Threads.nthreads()  ccwl.cpt[i].activehypo = vars; end
       
       addEntr = view(ccwl.params[sfidx], allelements[count])
-      # dynamic estimate with user requested speadNH of how much noise to inject (inflation or nullhypo)
-      spreadDist = calcVariableDistanceExpectedFractional(ccwl, sfidx, certainidx, kappa=ccwl.inflation)
       
       # do proposal inflation step, see #1051
       # consider duplicate convolution approximations for inflation off-zero
       # ultimately set by dfg.params.inflateCycles
       for iflc in 1:inflateCycles
+        # dynamic estimate with user requested speadNH of how much noise to inject (inflation or nullhypo)
+        spreadDist = calcVariableDistanceExpectedFractional(ccwl, sfidx, certainidx, kappa=ccwl.inflation)
```

Comment:
I had it consistent over inflate cycles to ensure inflation stays strong rather than anneal down, are you trying to reduce the "noise" on range only results, or increase the inflation strength?  Perhaps just give a bit more detail on why this change?

Reply:
increase the inflation strength: I do not know what changed, but it doesn't get there with a reasonable amount of `inflateCycles` 
But at least we know the problem has something to do with inflation.

Reply:
If the variance is small to start with `spreadDist` is small and inflation takes a long time to "walk around the circle"

Reply:
thanks Johan you're right